Library/Homebrew/dev-cmd/create.rb

```diff
@@ -122,6 +122,9 @@ def create_cask(args:)
     end
 
     cask_path.atomic_write <<~RUBY
+      # Documentation: https://docs.brew.sh/Cask-Cookbook
+      #                https://docs.brew.sh/Adding-Software-to-Homebrew#cask-stanzas
+      # PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
```

Comment:
I love this! Makes it far easier to know where to find the write information (both for new contributors, but even as a reference for existing)

Reply:
Not sure of the best way of doing so, but if it doesn't already exist, this seems like a good thing for `brew audit` to check for/confirm as well.